haskala/modules/custom/haskala_migrate/includes/haskala_migrate.entity.inc

```diff
@@ -75,13 +75,23 @@ abstract class HaskalaMigration extends Migration {
       ->execute();
 
     // When not finding a person, try to search only for its German name.
-    if (empty($result['node']) && $bundle == 'person' && mb_strrchr($title, ' - ')) {
-      list($name) = explode(' - ', $title);
-      $query = new EntityFieldQuery();
-      $result = $query->entityCondition('entity_type', 'node')
-        ->propertyCondition('type', 'person')
-        ->propertyCondition('title', $name)
-        ->execute();
+    if (empty($result['node']) && $bundle == 'person') {
+      if (mb_strrchr($title, ' - ')) {
```

Comment:
After checking, the code is necessary for some cases in migrate production producers with full name that appear in short title in person CSV.